server/prometheus.go

```diff
@@ -0,0 +1,27 @@
+package server
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/prebid/prebid-server/config"
+	metricsconfig "github.com/prebid/prebid-server/pbsmetrics/config"
+	prometheusMetrics "github.com/prebid/prebid-server/pbsmetrics/prometheus"
+)
+
+func newPrometheusServer(cfg *config.Configuration, metrics *metricsconfig.DetailedMetricsEngine) *http.Server {
+	var proMetrics *prometheusMetrics.Metrics
+
+	proMetrics = metrics.PrometheusMetrics
+
+	if proMetrics == nil {
+		glog.Fatal("Prometheus metrics configured, but a Prometheus metrics engine was not found. Cannot set up a Prometheus listener.")
+	}
+	return &http.Server{
+		Addr:    cfg.Host + ":" + strconv.Itoa(cfg.Metrics.Prometheus.Port),
+		Handler: promhttp.HandlerFor(proMetrics.Registry, promhttp.HandlerOpts{}),
```

Comment:
I think `ErrorLog`, `MaxRequestsInFlight`, and `Timeout` are pretty important here... just so that we can diagnose errors and make sure that the prometheus mothership doesn't bury the server.

Reply:
Are you asking to add metrics to record the prometheus interface activity, or something else?

Reply:
They're properties you can set on `HandlerOpts`